ftml/test/bold-block-paragraph.json

```diff
@@ -0,0 +1,40 @@
+{
+    "input": "[[b]]Paragraph\n\nBold[[/b]]",
```

Comment:
What happens with inputs like `[[b]]\nText\n[[/b]]` and `[[b]]One\nTwo[[/b]]`?

Reply:
I imagine it would look like [`span-newlines.json`](https://github.com/scpwiki/wikijump/blob/develop/ftml/test/span-newlines.json).